source/_integrations/cloud.markdown

```diff
@@ -23,6 +23,8 @@ cloud:
 
 Documentation of further configuration possibilites are located at [NabuCasa](https://www.nabucasa.com/config/)
 
+**Warning:** Files in the `www` folder (`/local/` url), aren't protected by the Home Assistant authentication. Files stored in this folder, if the URL is known, can be accessed by anybody without authentication. For better security, you should place any sensitive files (like camera snapshots), in a subdirectory with a long, random name like `hjkhsnf342345523lkjshdfiu/my_camera_image.jpg`.
+
```

Comment:
This is not part of the `cloud` integration, this is part of the `http` integration. Where this is already documented. 

https://www.home-assistant.io/integrations/http#hosting-files

Reply:
I understand that. However, since HTTP is enabled by default on a new install (as is cloud), most users will simply never visit the HTTP integration page and they will never see the warning to begin with. With the rise in users using cloud-enabled voice assistants (and hopefully using Nabu Casa for that access :) ), adding the warning to the cloud integration page further gets the message out there that the /www directory is unprotected. I don't think more information (or sometimes repeating good information and warnings) is a bad thing.

Reply:
We cannot repeat everything everywhere. There are tons of more information and warning dialog that apply to "cloud" if you approach it that way.